src/features/home/components/modules/HomeModule.tsx

```diff
@@ -122,7 +74,23 @@ const UnmemoizedModule = ({
     )
   }
 
-  return <React.Fragment></React.Fragment>
+  if (
+    (isHighlightOfferModule(item) && !enableNewExclusivityBlock) ||
+    (isExclusivityModule(item) && enableNewExclusivityBlock)
```

Comment:
question (non-blocking): est-ce que c'est testé ?

Reply:
Oui ! C'était précedemment testé avec les tests `should not display old exclusivity module when feature flag is true` et `should not show highlight module when feature flag is false`